rskj-core/src/main/java/co/rsk/rpc/Web3RskImpl.java

```diff
@@ -84,7 +86,13 @@ public void mnr_submitBitcoinBlock(String bitcoinBlockHex) {
         System.arraycopy(coinbaseAsByteArray, rskTagPosition+ RskMiningConstants.RSK_TAG.length, blockHashForMergedMiningArray, 0, blockHashForMergedMiningArray.length);
         String blockHashForMergedMining = TypeConverter.toJsonHex(blockHashForMergedMiningArray);
 
-        minerServer.submitBitcoinBlock(blockHashForMergedMining, bitcoinBlock);
+        SubmitBlockResult result = minerServer.submitBitcoinBlock(blockHashForMergedMining, bitcoinBlock);
+
+        if("OK".equals(result.getStatus())) {
```

Comment:
I don't like this strcmp

Reply:
It's only use is there, I don't think that there is a need to use an enum or smth like that.
What do you suggest ?